### ISSUE TITLE
Firefox no longer accepts unsigned add ons.

### ISSUE THREAD

**Issue by cypherpunk** [EFForg/privacybadgerfirefox#541]
Your Privacy Badger add on is being blocked by Firefox because newer versions of Firefox (version 40) protect you against malware and browser hijackers by warning you against third-party add-ons that are not digitally signed and verified by Mozilla. Mozilla verifies and "signs" add-ons that follow a set of guidelines to ensure that users' information will not be stolen or manipulated. All add-ons hosted on addons.mozilla.org undergo this review process in order to be verified and signed. Add-ons hosted on other sites will need to follow the same guidelines in order to be signed by Mozilla.  Add-ons that change your browser's settings without your consent or steal your information have become increasingly common. Some add-ons add unwanted toolbars or buttons, change your search settings or inject ads or malware into your device. This article explains how add-on signing protects you against such threats. 

Add-on signing targets only malware and browser hijacking. It does not control or censor the content that you choose to see.

**Comment by albell** [comment]
This is crucial for adoption. FF42 (currently Developer Edition) blocks unsigned add-ons completely, unless a preference is set in about:config, which the user base PB is targeting can't be expected to do.

**Comment by cooperq** [comment]
duplicate of #358
